app/__init__.py

```diff
@@ -20,7 +20,6 @@
 if app.config.get('TIMESYNC_URL') == 'http://timesync.example.com/':
     print 'ERROR: Configuration option "TIMESYNC_URL" not set'
     print '       Please set "TIMESYNC_URL" to the URL of your TimeSync server'
-    sys.exit(1)
```

Comment:
@subnomo
That doesn't work because `__init__.py` is the first file that is executed, so `app.config['TESTING']` will always be false. I'm not sure of the best way around this, so we might have to remove `sys.exit(1)`.